platform/core/src/services/DicomMetadataStore/DicomMetadataStore.js

```diff
@@ -211,7 +211,15 @@ const BaseImplementation = {
     let study = _getStudy(StudyInstanceUID);
     if (!study) {
       study = createStudyMetadata(StudyInstanceUID);
+      // Will typically be undefined with a compliant DICOMweb server, reset later
       study.StudyDescription = seriesSummaryMetadata[0].StudyDescription;
+      study.ModalitiesInStudy = [];
```

Comment:
First: `Modalities` is simple and representative enough, since it is accessed in the study (`study.Modalities`)

Second: You should add it to `createStudyMetadata` too

Reply:
ModalitiesInStudy in the DICOM name, and this is just recording the value when a study query isn't directly performed.  However it is good to add it to createStudyMetadata as well.